src/tools/merssh/cmakecommand.cpp

```diff
@@ -68,10 +71,30 @@ int CMakeCommand::execute()
         return 1;
     }
 
+    // FIXME If merssh is passed "'Foo Bar'", Command::arguments() returns ["'Foo", "Bar'"].
+    const bool abortOnMeta = false;
```

Comment:
abortOnMeta is used only once, and it's passed by value. I would just omit the whole variable.

Reply:
Compare `foo(false)` with `foo(abortOnMeta=false)` from reader's point of view :)